.travis.yml

```diff
@@ -1,46 +1,27 @@
 sudo: false
-
 language: python
-
 python:
-  - 2.7
-  - 3.5
-  - 3.6
-
+- 3.5
+- 3.8
 env:
-  - TOXENV=django111
-
-matrix:
- include:
-  - python: 3.5
-    env: TOXENV=quality
-  - python: 3.5
-    env: TOXENV=django20
-  - python: 3.5
-    env: TOXENV=django21
-  - python: 3.5
-    env: TOXENV=django22
-
+  - TOXENV=django22
+  - TOXENV=quality
 cache:
-  - pip
-
+- pip
 install:
-  - pip install -U pip wheel codecov
-  - pip install -r requirements/test.txt
-
+- pip install -U pip wheel codecov
+- pip install -r requirements/test.txt
 script:
-  - make test
-
+- make test
 after_success:
-  - codecov
-
+- codecov
 deploy:
   provider: pypi
   user: edx
   password:
     secure: xtak5/wlkNIWtIrY1kX9ht1xnL2Q7wMVHPofY2kEazILN8DJbwSqfNvwDV2E7mdevJK17o6dI1HDz7ZzzDAQaKudkN3tFIc/jZB9Cmxi/497NzOrhzshISCpnr+66r2+lR3bc2qsYH0W9w1LbVHzfWYYS4KhxBfCvSTA6mpxcwkgc38auoecnYa/SbqTgHhofmxg03E+Qb3czjoiloilS6sKNSuQdAbYB3nWVv/DsvDADkPSkqCXJh3J7dSrQS3oD9RHOg8Y5Rgzhqq/E2jIeTTU2lvaSplmkVBIy2jqi4Iys99a8hUPS6M9+ihsnn51znz58JJPUYZiMlQ5kiPzspMze0xhAD5ALpqCpvcJssFPumw4+M4MOcDZPdfzDlfEJne+IVP2mS7FJX2Tm1UWjlz/6vS+3DMJ6Xc0357GvjHcAmU8DaXep61n5jTapp5z/hZJcyYd+CsCDsvX3/Tl4u2dUTTFtHoYjRSq/gJ7kxgJYTDPEFMylN12IywSE5m2kPllFQCYT++2pgPlHuDbekpKhhTZ1iCmxjhvQ0FuqkGs1qkSftVtgY30sEpIMLSgiWJ7x3KMnmJUAh6bNyBmCUgFuEqU8+BRCF5C/hUrR5nKBmcroQECst2l6IgoXCMZOt8EAm2lIqKojANrvJmMVsNCd4thvJGdZ+TayG8b1Rc=
   distributions: sdist bdist_wheel
-  on:
+  true:
```

Comment:
Can you recheck that this is fine? @mraarif

Reply:
updated.